### PR TITLE
Greatly increases the chance for lavaland shiny mobs

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

title

## Why It's Good For The Game

they're wayyyyy too rare at the moment - this gives them a bigger chance.

## Changelog
:cl:
balance: Lavaland shiny mobs are now way more common.
balance: Goldgrubs now give out more mineral loot.
balance: Every fauna spawned now has a chance to be shiny, no matter how or why it spawned.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm [hunk 1]
```diff
@@ -111,15 +111,17 @@
 			qdel(bait)
 			visible_message("<span class='notice'>[src] examines [bait] closer, and telekinetically shatters the pen.</span>")
 
-/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/random/Initialize()
+/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/Initialize() //skyrat edit - every watcher has a chance to be a shiny
 	. = ..()
-	if(prob(1))
-		if(prob(75))
+	if(prob(10)) //skyrat edit
```
Comment (jjpark-kb) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm:116]: why is this 10 when goliath is 5.

Comment (BobJoga) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm:116]: design choice, thought it'd make sense for these to be less rare than the ancient goliath

Comment (BobJoga) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm]: i have conceded

**Comment by Azarak** [comment]
If they're no longer rare, then they're no longer special

**Pull request closed without merging**